### ISSUE TITLE
TestSniffer.test_sniffer fails with newer Python3

### ISSUE THREAD

**Issue by frispete** [wireservice/agate#746]
Hi,

we're experiencing [failed](https://build.opensuse.org/package/live_build_log/devel:languages:python:numeric/python-agate/openSUSE_Tumbleweed/x86_64) builds for openSUSE Tumbleweed, while a Python 3.6 build [succeeds](https://build.opensuse.org/package/live_build_log/devel:languages:python:numeric/python-agate/openSUSE_Leap_15.2/x86_64):
```
[    5s] =================================== FAILURES ===================================
[    5s] ___________________________ TestSniffer.test_sniffer ___________________________
[    5s] 
[    5s] self = <tests.test_py3.TestSniffer testMethod=test_sniffer>
[    5s] 
[    5s]     def test_sniffer(self):
[    5s]         with open('examples/test.csv') as f:
[    5s]             contents = f.read()
[    5s] >           self.assertEqual(csv_py3.Sniffer().sniff(contents).__dict__, csv.Sniffer().sniff(contents).__dict__)
[    5s] E           AssertionError: mappingproxy({}) != mappingproxy({'__module__': 'csv', '_name': 'sniffed', [137 chars]lse})
[    5s] 
[    5s] tests/test_py3.py:253: AssertionError
```

While at it, here is a warning, that might fail soon:
```
[    5s] =============================== warnings summary ===============================
[    5s] agate/utils.py:8
[    5s]   /home/abuild/rpmbuild/BUILD/agate-1.6.1/agate/utils.py:8: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
```
Would be nice, if somebody cares..

**Comment by bnavigator** [comment]
Bump.

**Comment by jpmckinney** [comment]
This was fixed in #737. A new release will be made shortly.
